### PR TITLE
OTPL-7842 Migrate parent to `basepom-minimal` directly

### PR DESCRIPTION
**Pull request by dkaukov**
inline OSS settings.

Same as we did for `eda-messages-java`

### CODE REVIEW

File: src/main/java/com/opentable/RuntimeMojoExecutionException.java [hunk 1]
```diff
@@ -25,11 +25,11 @@ public class RuntimeMojoExecutionException extends RuntimeException {
     private final MojoExecutionException mojoExecutionException;
 
     public RuntimeMojoExecutionException(MojoExecutionException mojoExecutionException) {
-        this.mojoExecutionException = mojoExecutionException;
+        this.mojoExecutionException = new MojoExecutionException(mojoExecutionException);
```
Comment (dkaukov) [inline src/main/java/com/opentable/RuntimeMojoExecutionException.java:28]: findbugs

File: src/main/java/com/opentable/RuntimeMojoExecutionException.java [hunk 2]
```diff
@@ -25,11 +25,11 @@ public class RuntimeMojoExecutionException extends RuntimeException {
...
     }
 
     public MojoExecutionException getMojoExecutionException() {
-        return mojoExecutionException;
+        return new MojoExecutionException(mojoExecutionException);
```
Comment (dkaukov) [inline src/main/java/com/opentable/RuntimeMojoExecutionException.java:32]: findbugs

**Review by mikebell90 — CHANGES_REQUESTED** [review]
please compare to otj-pg-embedded pr (which you have not reviewed btw) - these don't look that consistent

**Review by mikebell90 — APPROVED** [review]
This is fine but **PLEASE compare to otj-pg-embedded and review that PR too**

**Pull request merged**